### ISSUE TITLE
implement the fuck command 

### ISSUE THREAD

**Issue by p0W3RH311** [PowerShell/PowerShell#14696]
Hi Guys !

for better experience with console/terminal ... i suggest to implement new command "fuck" to renforce the suggestion mode

![example](https://user-images.githubusercontent.com/36144546/106649862-04a5f300-6547-11eb-8a3c-e7c361c93978.gif)

**Comment by rhymeswithmogul** [comment]
This might be present a localization issue for non-English users, due to the varying synonyms people may use for this command.  The only other issue I see is that the alias "fudge" should be defined for the benefit of users who build PowerShell with the `--disable-offensive` compile-time flag.

**Comment by SteveL-MSFT** [comment]
PowerShell DOES have a suggestion framework, but it needs to be fixed to work correctly.  I think the intent of `fuck` makes sense particularly when it's clear what the typo was supposed to do.  However, it seems we should have a cmdlet for it and a shorter alias (probably something more professional...).

**Comment by SeeminglyScience** [comment]
Worth noting that a community implementation does exist ([PoShFuck](https://github.com/mattparkes/PoShFuck)) though it does not use the suggestions framework.

**Comment by daxian-dbw** [comment]
I agree with Steve that a flexible suggestion framework would be the way to go. /cc @theJasonHelmick to comment.

**Comment by ThomasNieto** [comment]
@StevenBucher98 I believe this issue is addressed with feedback providers and predictors to choose and execute. However, there should also be a cmdlet created like Steve mentions to execute a suggestion `Invoke-Suggestion` with a short alias maybe something like `isug`.

**Comment by StevenBucher98** [comment]
@p0W3RH311 we do have some _similar_ to the experience you are showing in the GIF, check out https://devblogs.microsoft.com/powershell/what-are-feedback-providers/ and https://github.com/PowerShell/command-not-found which cover at least the apt get cases.

@ThomasNieto yeah it could be really interesting to do a cmdlet to invoke suggestions, I am little unsure about how it would integrate with feedback providers as currently feedback providers just trigger when their specific criteria is met. I wonder if we could port the rules from `fuck` to a feedback provider that will trigger on those errors but immediately instead of needing to type out the cmd. cc @daxian-dbw cause you may find this idea interesting 😄
